tracekit.js

```diff
@@ -697,7 +697,7 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
         }
 
         var chrome = /^\s*at (.*?) ?\(((?:file|https?|blob|chrome-extension|native|eval).*?)(?::(\d+))?(?::(\d+))?\)?\s*$/i,
-            gecko = /^\s*(.*?)(?:\((.*?)\))?(?:^|@)((?:file|https?|blob|chrome|\[).*?)(?::(\d+))?(?::(\d+))?\s*$/i,
+            gecko = /^\s*(.*?)(?:\((.*?)\))?(?:^|@)((?:file|https?|blob|chrome|\[native).*?)(?::(\d+))?(?::(\d+))?\s*$/i,
```

Comment:
> I'm worried that this could be localized. 

Searching through the [Firefox source code](https://dxr.mozilla.org/mozilla-central/search?q=%5Bnative&redirect=true&case=false), I don't think this string is localized.

Just to be sure, I also downloaded the [French edition of Firefox](https://www.mozilla.org/en-US/firefox/all/) and verified that the string is the same:

![image](https://cloud.githubusercontent.com/assets/2153/13962584/e0b96e74-f01e-11e5-9400-8d127acad49c.png)